### ISSUE TITLE
Check character encoding

### ISSUE THREAD

**Issue by sftsk** [anttiviljami/wp-server-log-viewer#1]
<img width="950" alt="screen shot 2016-08-30 at 15 26 17" src="https://cloud.githubusercontent.com/assets/646910/18088954/2a65211e-6ec6-11e6-9746-beba1f892679.png">

Seems like Umlauts aren't parsed the right way

**Comment by anttiviljami** [comment]
Which file are you reading this from ? When reading this from the terminal, does it show the umlauts correctly?

**Comment by sftsk** [comment]
Actually I don't know which file, but I think it's the default wp-palvelu 502 page. Just had this on the top of my log when I've first tried.

**Comment by anttiviljami** [comment]
Yeah, it does look like it. It also depends on whether it was written to the log properly in UTF-8 format. To me it looks like the log viewer is displaying the two byte character as two different chars.
